vqsim/vqmain.c

```diff
@@ -395,7 +411,7 @@ static int nodes_in_partition(int part)
 	}
 	return partnodes;
 }
-
+*/
```

Comment:
Might as well delete this function if it's not needed any more

Reply:
Removed